### DISCUSSION TITLE
Add keywords to CSL JSON

### DISCUSSION THREAD

**Discussion by sjgknight** [retorquere/zotero-better-bibtex#2725] [Q&A]
I can see from the [postscript description](https://retorque.re/zotero-better-bibtex/exporting/scripting/) and some [old zotero forum post](https://forums.zotero.org/discussion/63995/csl-json-export) that keywords can be added to the `Better CSL JSON` export, but I can't see how, I've tried these postscripts, which I hope are right direction(?) but don't get it. Any advice much appreciated!

```
if (Translator.BetterCSL) {
  csl.add({ name: 'keywords', value: zotero.tags });
  keywords = item.tags
}
```

**Comment by retorquere** [comment]
```javascript
if (Translator.BetterCSL) {
  if (item.tags) csl.keywords = item.tags.map(tag => tag.tag)
}
```

**Reply by sjgknight** [reply]
So easy when you know how! Thanks so much
